src/test/java/pl/project13/maven/git/GitCommitIdMojoIntegrationTest.java

```diff
@@ -176,7 +176,12 @@ public void shouldGenerateCustomPropertiesFileProperties(boolean useNativeGit) t
     // given
     mavenSandbox.withParentProject("my-pom-project", "pom")
                 .withChildProject("my-jar-module", "jar")
-                .withGitRepoInChild(AvailableGitTestRepo.GIT_COMMIT_ID)
+                // I feel there could be a mistake in the AvailableGitTestRepo's enum values
+                // the MAVEN_GIT_COMMIT_ID_PLUGIN has the same directory as WITH_ONE_COMMIT_WITH_SPECIAL_CHARACTERS
+                // I think the GIT_COMMIT_ID has the value which should be planned to be the plugin repo itself
```

Comment:
please remove this comment, the value is OK, simply the same repo can fulfil different purposes AFAIR.